### PR TITLE
Define imagem padrão para o perfil

### PR DESCRIPTION
**Pull request by gabriel-murakami**
mini PR na hora do almoço

### CODE REVIEW

**Comment by arturcp** [comment]
❤️

File: app/views/students/show.html.erb [hunk 1]
```diff
@@ -15,7 +15,11 @@
       <div class="card">
         <div class="card-body">
           <div class="d-flex flex-column align-items-center text-center">
-            <img src="<%= @student.avatar_src %>" width="150"/>
+            <% if @student.avatar %>
+              <img src="<%= @student.avatar_src %>" width="150"/>
```
Comment (arturcp) [inline app/views/students/show.html.erb]: maybe we could use image_tag here as well

**Review by arturcp — APPROVED** [review]

File: app/views/students/show.html.erb (hunk 1 above)
Comment (gabriel-murakami) [reply app/views/students/show.html.erb]: I tested it but it didn't work 🤔

Comment (gabriel-murakami) [reply app/views/students/show.html.erb]: never mind, I tested it wrong 😅

Comment (arturcp) [reply app/views/students/show.html.erb]: 😂

**Pull request merged**